include/deal.II/base/tensor_product_polynomials.h

```diff
@@ -79,6 +79,8 @@ class TensorProductPolynomials
   template <class Pol>
   TensorProductPolynomials (const std::vector<Pol> &pols);
 
+  virtual ~TensorProductPolynomials() {};
+
```

Comment:
Any reason why you need to make this class virtual? There is neither a virtual function added TensorProductPolynomialsBubble nor any use of TensorProductPolynomials as base class in some `delete` context as far as I can see.